lib/modules/platform/gerrit/readme.md

```diff
@@ -42,7 +42,7 @@ It works similar to the default option `"pr"`.
 
 You can use the `statusCheckNames` configuration to map any of the available branch checks (like `minimumReleaseAge`, `mergeConfidence`, and so on) to a Gerrit label.
 
-For example, if you want to use the [Merge Confidence](https://docs.renovatebot.com/merge-confidence/) feature and map the result of the Merge Confidence check to your Gerrit label "Renovate-Merge-Confidence" you can configure:
+For example, if you want to use the [Merge Confidence](../../../merge-confidence.md) feature and map the result of the Merge Confidence check to your Gerrit label "Renovate-Merge-Confidence" you can configure:
```

Comment:
@HonkingGoose so this url is totally wrong? that's why I thought it's generated from docs repo. 🤔

where should It point to now?

Reply:
The link should point to: [`docs/usage/merge-confidence.md`](https://github.com/renovatebot/renovate/blob/main/docs/usage/merge-confidence.md)